### PR TITLE
Organisation APIs

### PR DESCRIPTION
**Pull request by FsherinP**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by SMYALTAMASH** [comment]
@FsherinP can we close this PR if it need not be merged?

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

File: apis/orgapi/orgapi.yaml [hunk 1]
```diff
@@ -2035,21 +1339,72 @@ components:
...
+          err:
+          status: success
+          errmsg:
+        responseCode: OK
+        result:
+          response:
+            count: 1
+            content:
+              - keys: {}
+                channel": dikshaChannelNew
```
Comment (Sunitashelar) [inline apis/orgapi/orgapi.yaml:1386]: @FsherinP @mohitga we cannot add 'Diksha' specific terms in Sunbird. please make this generic. I am also concerned about the specific numeric keys given in the examples. Is this right?

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

File: apis/orgapi/orgapi.yaml (hunk 1 above)
Comment (mohitga) [reply apis/orgapi/orgapi.yaml:1386]: @FsherinP remove the  DIKSHA from channel keyword
@Sunitashelar  numeric is message-id, should be fine.  @FsherinP please confirm once or change the message-id

**Comment by keshavprasadms** [comment]
Hello @FsherinP,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://739.qa.docs.sunbird.org/3.8.0/ .<br/> **Note:** Only _3.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://739.qa.docs.sunbird.org/3.8.0/apis/orgapi// <br />  <br/> **Modified sections are -** <br />

File: apis/orgapi/orgapi.yaml (hunk 1 above)
Comment (FsherinP) [reply apis/orgapi/orgapi.yaml]: Did the changes

**Review by Sunitashelar — APPROVED** [review]

**Pull request merged**